egs2/README.md

```diff
@@ -61,6 +61,7 @@ See: https://espnet.github.io/espnet/espnet2_tutorial.html#recipes-using-espnet2
 | mini_librispeech        | Mini version of Librispeech corpus                                                                                               | DIAR                    | ENG                  | https://openslr.org/31/                                                                                      |              |
 | ml_openslr63            | Crowdsourced high-quality Malayalam multi-speaker speech data                           | ASR                     | MAL                  | https://openslr.org/63/                                                                                      |              |
 | mls                     | MLS (A large multilingual corpus derived from LibriVox audiobooks)                                                               | ASR                     | 8 languages          | http://www.openslr.org/94/                                                                                   |              |
+| msropenin               | Microsoft Speech Corpus (Indian languages) (A phrasal speech training and test data)    | ASR                     | TEL-GUJ-TAM langu | https://msropendata.com/datasets/7230b4b1-912d-400e-be58-f84e0512985e                                         |              |
```

Comment:
We might already have similar recipes in egs2. Can you check it and merge yours? https://github.com/espnet/espnet/tree/master/egs2/ms_indic_18/asr1